lib/stat_tracker.rb

```diff
@@ -0,0 +1,38 @@
+require 'CSV'
+# require './lib/game.rb'
+# require './lib/team.rb'
+# require './lib/game_teams.rb'
```

Comment:
Once these files and their corresponding classes have been created, these lines will need to be commented back in.